static/app/views/issueDetails/groupTags/groupTagValues.tsx

```diff
@@ -399,10 +401,9 @@ const StyledExternalLink = styled(ExternalLink)`
   margin-left: ${space(0.5)};
 `;
 
-const Column = styled('div')`
-  display: flex;
-  align-items: center;
-`;
+function Column(props: FlexProps) {
+  return <Flex align="center" {...props} />;
+}
```

Comment:
Inline

Reply:
but the only usages of `Column` are `styled(Column)` :/ I think we should probably just remove the `Column` abstraction and add `align="center"` to the places